app/models/local_authority.rb

```diff
@@ -36,10 +36,10 @@ def create_la_funded_places!(urn:, device_allocation: 0, router_allocation: 0, e
       postcode: postcode,
     }.reverse_merge(extra_args)
 
-    funded_places = LaFundedPlace.create!(attrs)
-    funded_places.create_preorder_information!(who_will_order_devices: 'responsible_body', will_need_chromebooks: 'no')
-    funded_places.device_allocations.std_device.create!(allocation: device_allocation)
-    funded_places.device_allocations.coms_device.create!(allocation: router_allocation)
-    funded_places
+    funded_place = LaFundedPlace.create!(attrs)
+    funded_place.create_preorder_information!(who_will_order_devices: 'responsible_body')
```

Comment:
not part of this change but are schools not ordering? otherwise we have to attach users to RB who will have access to everything?

Reply:
Yes, it should be schools now, I think originally we thought it would be the RB, I'll change that and make sure it all still works 😬

Reply:
Change made and seems to work fine when set up as `school` so far.